api/v1alpha1/contour_types.go

```diff
@@ -91,6 +91,14 @@ type ContourSpec struct {
 	// +optional
 	GatewayClassRef *string `json:"gatewayClassRef,omitempty"`
```

Comment:
We can deprecate the `gatewayClassRef` now as well. The `controllerName` is used for determining the gatewayclass.

Reply:
I need to dig more into this, but one small comment if you want to tackle here or in another PR.

Reply:
Yeah I wasn’t sure whether we could just remove it now or not bc of API versioning, do we need to do a v1alpha2 to remove it? Or just add a comment/annotation if we can